draft-bonaventure-mptcp-converters.mkd

```diff
@@ -671,6 +685,14 @@ own value. For the TCP Options that are included in the 'TCP Options' field with
 an optional value, it shall copy the entire option for use in the connection with the destination peer. This feature is required to support TCP Fast Open.
 
 
+### External Connect TLV {#sec-extrnl-connect}
```

Comment:
Can we change the name to "Listen" to make it explicit that the host is asking the Converter to listen for incoming connections that will be relayed into MPTCP ones.